google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/spi/v1/PublisherImplTest.java

```diff
@@ -362,11 +362,13 @@ public void testPublisherGetters() throws Exception {
     Publisher publisher = builder.build();
 
     assertEquals(TEST_TOPIC.toString(), publisher.getTopic());
-    assertEquals(10, publisher.getMaxBundleBytes());
-    assertEquals(new Duration(11), publisher.getMaxBundleDuration());
-    assertEquals(12, publisher.getMaxBundleMessages());
-    assertEquals(Optional.of(13), publisher.getMaxOutstandingRequestBytes());
-    assertEquals(Optional.of(14), publisher.getMaxOutstandingElementCount());
+    assertEquals(10, (long) publisher.getBundlingSettings().getRequestByteThreshold());
```

Comment:
Try not to change the surface next time you do this kind of transformation - it's really hard to separate out what is changed and what isn't. It's fine this time if it is only confined to this particular change (bundling settings & flow control settings).